### PR TITLE
GH-237: Extended the troubleshooting section for Windows.

### PR DESCRIPTION
**Pull request by kittaakos**
Task: #237 

@akosyakov, please review. 

Signed-off-by: Akos Kitta <kittaakos@gmail.com>

### CODE REVIEW

File: doc/Developing.md [hunk 1]
```diff
@@ -198,6 +198,7 @@ Theia uses native modules and also requires Python 2.x to be installed on the sy
  - You don't have write access to the installation directory.
  - Try to run your command line (`PowerShell`, `GitBash`, `Cygwin` or whatever you are using) as an administrator.
  - The permissions in the NPM cache might get corrupted. Please try to run `npm cache clean` to fix them.
+ If you experience issues such as `Error: EBUSY: resource busy or locked, rename`, try to disable (or uninstall) your anti-malware software. See [here](https://github.com/npm/npm/issues/13461#issuecomment-282556281).
```
Comment (akosyakov) [inline doc/Developing.md]: should it be a bullet point?

Comment (kittaakos) [inline doc/Developing.md]: I thought about that but it is unrelated to the `eperm` issues from above.

Comment (kittaakos) [inline doc/Developing.md]: But it looks bad. You're right.

Comment (akosyakov) [inline doc/Developing.md]: then `Still having issues on Windows?` should be a new paragraph as well?
